includes/class-assets.php

```diff
@@ -267,8 +267,8 @@ public function enqueue_css_override() {
      *
      * @param bool $force Force reloading the resources.
      */
-	public function enqueue_admin_scripts( $force = false ) {
-		if ( ! $force && ! WPBDP_App_Helper::is_bd_page() ) {
```

Comment:
We still may need to keep the force option for reverse compatibility.

Reply:
The `$force` would bring issues in later on. The [admin_enqueue_scripts](https://developer.wordpress.org/reference/hooks/admin_enqueue_scripts/) uses the string param `$hook_suffix` . We need to only load resources for BD pages

Reply:
There might be pages for an add-on that aren't counted as a Bd page. If there are cases like that, this would break the add-on (or a custom add-on)

Reply:
This should still work fine for an empty string:
if ( ! $force && ! WPBDP_App_Helper::is_bd_page() ) {

Reply:
I updated the message so we can merge this.